src/guides/v2.3/install-gde/system-requirements-tech.md

```diff
@@ -36,23 +36,22 @@ Upgrading the Magento applications and extensions you obtain from Magento Market
 
 ## Database
 
-MySQL 5.6, 5.7
+*  MySQL 5.7 or on-premise installations
+*  MariaDB 10.2 for {{site.data.var.ece}} projects
```

Comment:
For Cloud, the versions table shows MariaDB 10.2, 10.3, and 10.4

Reply:
Product Management requested this change

Reply:
Ignore me-- 10.3 and 10.4 are for 2.4.